pyproject.toml

```diff
@@ -34,6 +46,160 @@ norecursedirs = [
     "kernel"
 ]
 
+[tool.coverage.run]
+omit = [
+    "*/migrations/*",
+    "kernel/*",
+    "*/apps.py",
+    "manage.py",
+]
+
+[tool.coverage.report]
+exclude_lines = [
+    "pragma: no cover",
+    "def __repr__",
+    "if self\\.debug",
+    "raise AssertionError",
+    "if 0:",
+    "if __name__ == .__main__.:"
+]
+
+[tool.ruff]
+line-length = 88
+exclude = [
+    "venv/*",
+    ".venv/*",
+    "build/*",
+    "dist/*",
+    ".git/*",
+    "__pycache__/*",
+    "*.egg-info/*",
+    ".mypy_cache/*",
+    ".pytest_cache/*",
+    "migrations/*"
+]
+
+[tool.ruff.lint]
+ignore = [
+    "E203",  # Ignore whitespace before ':', ';', or '#'
+    "E501"   # Ignore line length issues (lines longer than 88 characters)
+]
+select = [
+    "E",     # Select all PEP8 error codes
+    "W",     # Select all warnings
+    "F",     # Select all potential runtime errors
+    "C90"    # Select custom or extended error code C90
+]
+
+[tool.mypy]
+mypy_path = "stubs"
+disallow_untyped_calls = true
+disallow_untyped_defs = true
+ignore_missing_imports = true
+explicit_package_bases = true
+exclude = '''
+^docs/source/conf.py|
+^build/|
+^tests/|
+^stubs/|
+^kernel/
+'''
+
+[tool.black]
+line-length = 88
+exclude = '''
+/(
+    \.git
+  | \.venv
+  | build
+  | dist
+  | migrations
+  | venv
+  | env
+  | __pycache__
+  | static
+  | media
+  | node_modules
+  | env
+  | kernel
+  | \.mypy_cache
+  | \.pytest_cache
+  | .*\.egg-info
+)/
+'''
+
+[tool.isort]
+profile = "black"
+line_length = 88
+skip = [
+    "venv",
+    ".venv",
+    "build",
+    "dist",
+    ".git",
+    "__pycache__",
+    "*.egg-info",
+    ".mypy_cache",
+    ".pytest_cache",
+    "migrations",
+    "static",
+    "media",
+    "node_modules",
+    "env",
+    "kernel"
+]
+
+[tool.commitizen]
+name = "cz_conventional_commits"
+version = "0.1.3"
```

Comment:
fix the version! start with 0.1.0.

read about cz bump and changelog rules

@ARYAN-NIKNEZHAD